OpenRA.Game/Settings.cs

```diff
@@ -171,6 +171,9 @@ public class GraphicSettings
 		[Desc("Disable separate OpenGL render thread on Windows operating systems.")]
 		public bool DisableWindowsRenderThread = true;
 
+		[Desc("Disable separate OpenGL render thread on Windows operating systems.")]
```

Comment:
Copy-paste from above, is that intentional?

Reply:
No. Fixed.